assets/js/modules/analytics/components/dashboard/index.js

```diff
@@ -34,3 +34,8 @@ export { default as LegacyAnalyticsDashboardWidgetPopularPagesTable } from './Le
 export { default as LegacyAnalyticsDashboardWidgetTopAcquisitionSources } from './LegacyAnalyticsDashboardWidgetTopAcquisitionSources';
 export { default as LegacyAnalyticsDashboardWidgetTopLevel } from './LegacyAnalyticsDashboardWidgetTopLevel';
 export { default as LegacyDashboardAcquisitionPieChart } from './LegacyDashboardAcquisitionPieChart';
+export { default as DashboardAllTrafficWidgetV2 } from './DashboardAllTrafficWidgetV2';
+export { default as DimensionTabs } from './DimensionTabs';
+export { default as TotalUserCount } from './TotalUserCount';
+export { default as UserCountGraph } from './UserCountGraph';
+export { default as UserDimensionsPieChart } from './UserDimensionsPieChart';
```

Comment:
@kostyalmm let's move new files to the `DashboardAllTrafficWidgetV2` folder and rename the `DashboardAllTrafficWidgetV2` file to be just `index.js`. Then we will need to re-export only `DashboardAllTrafficWidgetV2` here like this:

```js
export { default as DashboardAllTrafficWidgetV2 } from './DashboardAllTrafficWidgetV2';
```

It should solve the problem with that javascript test. If it doesn't help ping me in slack.

Reply:
worked

Reply:
@kostyalmm have you pushed your changes? I don't see it.